recipes-kernel/linux/files/patches-6.1/0083-dts-iot2050-Support-IOT2050-SM-variant.patch

```diff
@@ -0,0 +1,383 @@
++	/delete-property/ pinctrl-50;
++
++	user1_led_pins: user1-led-default-pins {
++		pinctrl-single,pins = <
++			/* (AB1) WKUP_UART0_RXD:WKUP_GPIO0_52, as USER 1 led red */
++			AM65X_WKUP_IOPAD(0x00a0, PIN_OUTPUT, 7)
++			/* (AB5) WKUP_UART0_TXD:WKUP_GPIO0_53, as USER 1 led green */
++			AM65X_WKUP_IOPAD(0x00a4, PIN_OUTPUT, 7)
++		>;
++	};
++
++	soc_asic_pins: soc-asic-default-pins {
++		pinctrl-single,pins = <
++			AM65X_WKUP_IOPAD(0x0044, PIN_INPUT, 7)  /* (P4) WKUP_GPIO0_29 */
++			AM65X_WKUP_IOPAD(0x0048, PIN_INPUT, 7)  /* (P5) WKUP_GPIO0_30 */
++			AM65X_WKUP_IOPAD(0x004c, PIN_INPUT, 7)  /* (P1) WKUP_GPIO0_31 */
++		>;
++	};
++};
++
++&main_gpio0 {
++	gpio-line-names = "main_gpio0-base";
++};
++
++&main_gpio1 {
++	pinctrl-names = "default";
++	pinctrl-0 =
++		<&cp2102n_reset_pin_default>,
++		<&main_pcie_enable_pins_default>,
++		<&asic_spi_mux_ctrl_pin>;
++	gpio-line-names =
++		/* 0..9 */
++		"", "", "", "", "", "", "", "", "", "",
++		/* 10..19 */
++		"", "", "", "", "", "", "", "", "", "",
++		/* 20..29 */
++		"", "", "", "", "CP2102N-RESET", "", "", "", "", "",
++		/* 30..39 */
++		"", "", "", "", "", "", "", "", "", "",
++		/* 40..49 */
++		"", "", "", "", "", "", "", "", "", "",
++		/* 50..59 */
++		"", "", "", "", "", "", "", "", "", "",
++		/* 60..69 */
++		"", "", "", "", "", "", "", "", "", "",
++		/* 70..79 */
++		"", "", "", "", "", "", "", "", "", "",
++		/* 80..86 */
++		"", "", "", "", "", "", "ASIC-spi-mux-ctrl";
++};
++
++&wkup_gpio0 {
++	pinctrl-names = "default";
++	pinctrl-0 =
++		<&push_button_pins_default>,
++		<&db9_com_mode_pins_default>,
++		<&soc_asic_pins>;
++	gpio-line-names =
++		/* 0..9 */
++		"wkup_gpio0-base", "", "", "", "UART0-mode1", "UART0-mode0",
++		"UART0-enable", "UART0-terminate", "", "WIFI-disable",
++		/* 10..19 */
++		"", "", "", "", "", "", "", "", "", "",
++		/* 20..29 */
++		"", "", "", "", "", "USER-button", "", "", "","ASIC-gpio-0",
++		/* 30..31 */
++		"ASIC-gpio-1", "ASIC-gpio-2";
++};
++
++&main_spi0 {
++	pinctrl-names = "default";
++	pinctrl-0 = <&main_spi0_pins>;
++
++	#address-cells = <1>;
++	#size-cells= <0>;
++
++	spidev@0 {
++		compatible = "rohm,dh2228fv";
++		spi-max-frequency = <20000000>;
++		reg = <0>;
++	};
++};
++
++&mcu_spi0 {
++	pinctrl-names = "default";
++	pinctrl-0 = <&mcu_spi0_pins_default>;
++};
++
++&main_i2c3 {
++	pinctrl-names = "default";
++	pinctrl-0 = <&main_i2c3_pins_default>;
++	clock-frequency = <400000>;
++
++	accelerometer: lsm6dso@6a {
++		compatible = "st,lsm6dso";
++		reg = <0x6a>;
++	};
++
++	lightsensor: pm16d17@44 {
++		compatible = "everlight,pm16d17";
++		reg = <0x44>;
++
++		ps-gain = <1>;
++		ps-itime = "0.4";
++		ps-wtime = "25";
++		ps-ir-led-pulse-count = <1>;
++	};
++
++	/delete-node/ edp-bridge@f;
++};
++
++/delete-node/ &pcal9535_1;
++/delete-node/ &pcal9535_2;
++/delete-node/ &pcal9535_3;
++
++&dss {
++	status = "disabled";
++};
++
++&dss_ports {
++	/delete-node/ port@1;
++};
++
++&ecap0 {
++	status = "disabled";
++};
```

Comment:
We should add https://lore.kernel.org/linux-arm-kernel/6131d44e0505ca3efbb9039e5f2b637a3e139312.1699087938.git.jan.kiszka@siemens.com/ to the queue instead, then this is obsolete.

Reply:
Not default disabled for 6.1... Anyway, to make it unified and reduce the future effort, let's remove it here and  since the upstream has already disabled it by default, it should be ok for upstreaming.

Also make it okay is harmless.